site/components/src/main/java/uk/nhs/digital/intranet/model/SearchResult.java

```diff
@@ -1,8 +1,10 @@
 package uk.nhs.digital.intranet.model;
 
-public interface SearchResult {
+import org.hippoecm.hst.content.beans.standard.HippoBean;
 
-    String getTitle();
+public interface SearchResult extends HippoBean {
```

Comment:
Why does the interface need to extend `HippoBean` ? Is this required for the aggregation of results into `pageable` ?

Reply:
Also, in this case, does `Person` need to extend `HippoItem` ?

Reply:
yes to point 1, for point 2, actually it could just implement the interface since we are overriding/implementing all the methods anyway